src/test/codegen/match.rs

```diff
@@ -13,17 +14,23 @@ pub fn exhaustive_match(e: E) -> u8 {
 // CHECK: switch{{.*}}, label %[[OTHERWISE:[a-zA-Z0-9_]+]] [
 // CHECK-NEXT: i[[TY:[0-9]+]] [[DISCR:[0-9]+]], label %[[A:[a-zA-Z0-9_]+]]
 // CHECK-NEXT: i[[TY:[0-9]+]] [[DISCR:[0-9]+]], label %[[B:[a-zA-Z0-9_]+]]
+// CHECK-NEXT: i[[TY:[0-9]+]] [[DISCR:[0-9]+]], label %[[C:[a-zA-Z0-9_]+]]
 // CHECK-NEXT: ]
 // CHECK: [[OTHERWISE]]:
 // CHECK-NEXT: unreachable
+//
 // CHECK: [[A]]:
 // CHECK-NEXT: store i8 0, {{i8\*|ptr}} %1, align 1
 // CHECK-NEXT: br label %[[EXIT:[a-zA-Z0-9_]+]]
 // CHECK: [[B]]:
 // CHECK-NEXT: store i8 1, {{i8\*|ptr}} %1, align 1
+// CHECK-NEXT: br label %[[EXIT:[a-zA-Z0-9_]+]]
```

Comment:
nit: this isn't really your problem, since you're clearly just following the existing pattern here, but it's odd that this is redefining the `EXIT` filecheck variable multiple times.

I think maybe this is intending to check that they're all going to the same place?  So consider changing it to something like
```
// CHECK: [[A]]:
// CHECK-NEXT: store i8 0, {{i8\*|ptr}} %1, align 1
// CHECK-NEXT: br label %[[EXIT:[a-zA-Z0-9_]+]]
// CHECK: [[B]]:
// CHECK-NEXT: store i8 1, {{i8\*|ptr}} %1, align 1
// CHECK-NEXT: br label %[[EXIT]]
// CHECK: [[C]]:
// CHECK-NEXT: store i8 2, {{i8\*|ptr}} %1, align 1
// CHECK-NEXT: br label %[[EXIT]]
// CHECK: [[EXIT]]:
```
(It [looks like that would pass](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=194c87b9ac7e7253af638778be6bb256).)

Reply:
Ok.